vars.tf

```diff
@@ -46,7 +46,10 @@ variable force_destroy {
 }
 
 variable databases {
-  type        = list(any)
+  type = list(object({
+    name      = string
+    dialect   = optional(string)
```

Comment:
Now you need to add the other optional values: ddl, version_retention_period etc